app/assets/stylesheets/application.scss

```diff
@@ -23,6 +23,7 @@ $govuk-use-legacy-palette: false;
 @import 'govuk_publishing_components/components/_layout-header';
 @import 'govuk_publishing_components/components/_layout-footer';
 @import 'govuk_publishing_components/components/_title';
+@import 'govuk_publishing_components/components/_search';
```

Comment:
This is the stylesheet for the search component, but I can't see the component being called in the code. Is it in a separate commit or already present somewhere?

Reply:
Ah, sorry, never mind then.